EventHorizon/Frame/MainFrame.lua

```diff
@@ -302,6 +302,11 @@ function MainFrame:UpdateAllFrames()
 			spellFrame
 			:GetIcon()
 			:SetSize(height, height)
+			
+			local coordScale = (EventHorizon.opt.zoom) / 100
```

Comment:
Need to set a default value for zoom in the default profile so that this does not cause an arithmetic error when the value is nil. Once you get that in, I will merge

Reply:
Good spot, thanks :)